### PR TITLE
Added type hints to ImageMath

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

File: src/PIL/Image.py [hunk 1]
```diff
@@ -1305,7 +1305,7 @@ def getbands(self):
         """
         return ImageMode.getmode(self.mode).bands
 
-    def getbbox(self, *, alpha_only=True):
+    def getbbox(self, *, alpha_only=True) -> tuple[int, int, int, int]:
```
Comment (hugovk) [inline src/PIL/Image.py:1308]:
```suggestion
    def getbbox(self, *, alpha_only: bool = True) -> tuple[int, int, int, int]:
```

File: src/PIL/ImageMath.py [hunk 2]
```diff
@@ -247,7 +257,7 @@ def eval(expression, _dict={}, **kw):
 
     compiled_code = compile(expression, "<string>", "eval")
 
-    def scan(code):
+    def scan(code) -> None:
```
Comment (hugovk) [inline src/PIL/ImageMath.py:260]: What type is `code`?

Comment (radarhere) [reply src/PIL/ImageMath.py]: I've pushed a commit.

Comment (hugovk) [reply src/PIL/ImageMath.py]: `code`'s type is `CodeType` :D

**Pull request merged**